### DISCUSSION TITLE
Add Bond Information

### DISCUSSION THREAD

**Discussion by maravento** [cockpit-project/cockpit#18400] [Q&A] [answered]
Page: "Network"

Where can I find detailed information on how the "add bond" option works?. (On how each option works in practice with examples)

I only found this link on oracle, but it's not enough information
https://docs.oracle.com/en/operating-systems/oracle-linux/cockpit/cockpit-network.html#configure-a-network-bond

Thanks in advance

**Comment by martinpitt** [comment, marked as answer]
Indeed the Oracle docs only explain the types and options of bonds, not their purpose.

If you open the "Help" menu, at least in Fedora/RHEL there is an entry "Managing networking bonds" which leads you to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/configuring-network-bonds-using-the-web-console_system-management-using-the-rhel-8-web-console . This has some further explanation and links.

**Reply by maravento** [reply]
I already checked and it is the same information from Oracle. In Oracle, at least, it is more specific when it comes to what each mode does and in RHEL it is more confusing, except in "active backup", which gives an example.
For example, this part is confusing: "The active-backup, balance-tlb, and balance-alb modes." I already know what is "active-backup" but what is "balance-tlby" and "balance-alb". The document does not clarify if it refers to Round Robin, Broadcast, XOR, etc. And I could expand, but to say that the document does not add anything new to what Oracle mentions is enough.
Thank you

**Reply by maravento** [reply]
I had not realized that cockpit does "link aggregation" and not "load balancing". Thanks and sorry for the confusion.
